config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-periodic-dra.yaml

```diff
@@ -323,3 +323,85 @@ periodics:
               value: "true"
             - name: NODE_PRELOAD_IMAGES
               value: "gcr.io/k8s-staging-perf-tests/sleep:v0.0.3"
+
+  - name: ci-kubernetes-e2e-gce-100-node-dra-extended-resources-with-workload
+    cluster: k8s-infra-prow-build
+    tags:
+      - "perfDashPrefix: gce-dra-extended-resources-100Nodes-with-workload"
+      - "perfDashBuildsCount: 270"
+      - "perfDashJobType: performance"
+    interval: 4h
```

Comment:
should we add a TODO to relax this aggressive interval after we've gotten sufficient mileage during the extended resources beta validation period?